features-json/cors.json

```diff
@@ -125,14 +125,14 @@
       "58":"y",
       "59":"y",
       "60":"y",
-      "61":"y #4",
-      "62":"y #4",
-      "63":"y #4",
-      "64":"y #4",
-      "65":"y #4",
-      "66":"y #4",
-      "67":"y #4",
-      "68":"y #4",
+      "61":"y #4 #5",
```

Comment:
This is a bit weird to me. Looks like 4 and 5 suddenly appeared at version 61.

The same goes to the other changes; 5 suddenly appeared at version X, then disappeared at version Y.